common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -165,13 +173,15 @@ def definition_from_xml(cls, xml_object, system):
         raise NotImplementedError("%s does not implement definition_from_xml" % cls.__name__)
 
     @classmethod
-    def clean_metadata_from_xml(cls, xml_object):
+    def clean_metadata_from_xml(cls, xml_object, excluded_fields=list()):
```

Comment:
Please make the `excluded_fields` default arg be `None`, or an empty tuple. This currently makes `excluded_fields` a global so that any side-effects on it will persist across invocations of the function. Granted, it's not being mutated in the function at present, but it's one of those unintuitive things that bites people much later down the road.